bridge/server/interfaces/webhook-config-yaml.ts

```diff
@@ -69,9 +75,15 @@ export class WebhookConfigYaml implements WebhookConfigYamlResult {
           requests: [curl],
           envFrom: secrets,
           subscriptionID: subscriptionId,
+          sendFinished: sendFinished,
         });
       } else {
-        this.spec.webhooks.push({ type: eventType, requests: [curl], subscriptionID: subscriptionId });
+        this.spec.webhooks.push({
+          type: eventType,
+          requests: [curl],
+          subscriptionID: subscriptionId,
+          sendFinished: sendFinished,
+        });
       }
     } else {
```

Comment:
The `sendFinished` property is not being updated when the webhook already exists